songs.py

```diff
@@ -29,6 +29,61 @@ def get_spotify_genres():
     return genres
 
 
+def get_similar(song, limit):
```

Comment:
I'm sure this is a work in progress but make sure to document/code this function.